IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_MemberTypeCatenaryStay/DocPropertySet.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_MemberTypeCatenaryStay" UniqueId="b15eff21-bd3f-df08-8b2a-e9722e73b14d" ApplicableType="IfcMember/BRACE" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="AssemblyInstruction_3W5i6nbkP9NBt_M6PTB5jW" />
```

Comment:
I believe we decided that properties like this are supposed to be grouped in "Installation" or "Maintainance" PSets

Reply:
AssemblyInstruction is the only one we keep on Pset. @czapplitec  has the information in case we associate it to a global pset. c0f5835a24d8f1939a80ada93c3b1cb04772002d